### ISSUE TITLE
springfox 2.2.2 no api docs are generated

### ISSUE THREAD

**Issue by justynaGithub** [springfox/springfox#1059]
Hello!

I have a problem integrating Springfox 2.2.2 into my Spring MVC project. I have just described the problem here: http://stackoverflow.com/questions/33780237/springfox-2-2-2-no-api-docs-generated .
Can you please take a look and help?

Kind regards, 
Justyna

**Comment by jerryxing98** [comment]
I have the same problem ! Anybody have some help?

**Comment by dilipkrish** [comment]
@justynaGithub @jerryxing98  problem is fixed? if not let me know

**Comment by rockytriton** [comment]
Looks like it's fixed per http://stackoverflow.com/a/34040828/428916

**Comment by dilipkrish** [comment]
Closing since it appears the solution is available Thanks @rockytriton :metal:

**Comment by mevinsh** [comment]
Hi
I am using springfox v2.4.0 and getting this exact same issue. I have already tried all different configurations above and nothing seems to be working. Also one other thing I noticed is that for Docket when I setup .apis(RequestHandlerSelectors.any()) then springfox goes into an infinite recursive scan and my server never startups (if i let it run then its eventually gets an OOM). If I use .apis(RequestHandlerSelectors.basePackage("x.y.z")) then I get a exception at runtime when I try to access v2/api-docs
I could only get my server up with .apis(RequestHandlerSelectors.none()) but I do not get any APIs when I visit v2/api-docs just the base information i setup in Swagger config
